R/optionalInput.R

```diff
@@ -1,18 +1,18 @@
-#' Hide, Show Label only or display a `pickerInput`
+#' Wrapper on `pickerInput`
 #'
 #' @description `r lifecycle::badge("stable")`
-#' Hidden input widgets are useful to have the `input[[inputId]]` variable
-#' on available in the server function but no corresponding visual clutter from
-#' input widgets that provide only a single choice.
+#' Wrapper on [shinyWidgets::pickerInput()] with additional features.
+#' When `fixed = TRUE` or when the number of `choices` is less or equal to 1,
+#' the `pickerInput` widget is hidden and non-interactive widget will be displayed
+#' instead. With change of the user interface `selected` remains unchanged.
+#' Toggle of `HTML` elements is just the visual effect to avoid displaying
+#' `pickerInput` widget when there is only one choice.
```

Comment:
```suggestion
#' Wrapper for [shinyWidgets::pickerInput()] with additional features.
#' When `fixed = TRUE` or when the number of `choices` is less than 2,
#' the `pickerInput` widget is hidden and a non-interactive widget is displayed
#' instead. 
#' Change of the user interface does not affect the selection `selected`.
#' Toggling of `HTML` elements is just a visual effect to avoid displaying the 
#' `pickerInput` widget when there is only one choice.
```